lib/live_view_native/swiftui/rules_parser/modifiers.ex

```diff
@@ -58,21 +58,21 @@ defmodule LiveViewNative.SwiftUI.RulesParser.Modifiers do
   end
 
   ime_function = fn is_initial ->
-    if is_initial do
-      empty()
-    else
-      ignore(string("."))
-    end
-    |> ignore(string("to_ime"))
+    # if is_initial do
+    #   empty()
+    # else
+    #   ignore(string("."))
+    # end
```

Comment:
> Can this be removed
Yes! Thank you.

---

> Would it be possible to have it do simple string interpolation ...

TLDR: I think the current approach is the best given the parser implementation we have, but I'm open to what else we should support.

It sounds like you want to be able to do the normal string interpolation so that something like the following could work in future:

```elixir
# the style
"h-" <> height do
  height(#{height})
end

# Using the class
<Button class="h-:infinity"> ... </Button>

#expected output
height(:infinity)
```

Unfortunately, I don't think that would work. The current model assumes that the variable part of the class name is known at runtime (_after_ we have already compiled the AST).

```mermaid
flowchart LR
    A(SHEET) -->|Compile Time| B(class function)
    B         -->|Runtime| C(AST)
    :infinity -->|Runtime| C(AST)
```

The purpose of the helper functions (`to_ime`, `to_float` and `to_integer`) was to allow us to map runtime strings into other types.

Let me know if you have any specific use cases, you'd have wanted to support, and we can extend this PR

Reply:
Interesting, I didn't realize those were evaluated at runtime. Thanks for the clarification.

Reply:
@carson-katri we're going to no use this syntax as we want to to reserve the Elixir interpolation for instances of dropping down into writing the functions by hand. We'll do what Heex has and use `{ ...  }` instead for interpolation within the sheet.

As far as executing Eixir functions, I don't think that's a good idea for us to do in the sheet because of the `attr` accessor. That value isn't accessible at compile-time or runtime but only when yanked from the element by the SwiftUI client.

In your examples above the recommendation for now is to write a custom `class/2` function if you more complex manipulations:

```elixir
def class("frame-" <> dims, _target) do
   [width, height] = String.split(dim, ":")

   ~RULES"""
   frame(width: #{width}, height: #{height}
   """
end
```